.github/config.yml

```
@@ -0,0 +1,21 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thank you for opening your first issue here! :tada: Be sure to follow the issue template :)
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Thank you for opening this pull request! We appreciate your contributions! Please check out our contributing guidelines.
```

```suggestion
  Thank you for opening this pull request! We appreciate your contributions! Please check out our [contributing guidelines](https://github.com/WorldHealthOrganization/open-source-communication-channel/blob/main/CONTRIBUTING.md).
```

Would be good to link to the contributing guidelines here?